wrappers/vroom.rb

```diff
@@ -267,8 +269,14 @@ def collect_jobs(vrp, vrp_skills, vrp_units)
             service.sticky_vehicles.flat_map{ |sticky| vrp_skills.find_index{ |sk| sk == sticky.id } }.compact,
           priority: (100 * (8 - service.priority).to_f / 8).to_i, # Scale from 0 to 100 (higher is more important)
           time_windows: service.activity.timewindows.map{ |timewindow| [timewindow.start, timewindow.end || 2**30] },
-          delivery: vrp_units.map{ |unit| service.quantities.find{ |quantity| quantity.unit.id == unit.id && quantity.value.negative? }&.value&.to_i || 0 },
-          pickup: vrp_units.map{ |unit| service.quantities.find{ |quantity| quantity.unit.id == unit.id && quantity.value.positive? }&.value&.to_i || 0 }
+          delivery: vrp_units.map{ |unit|
+            q = service.quantities.find{ |quantity| quantity.unit.id == unit.id && quantity.value.negative? }
+            ((q&.value || 0) * CUSTOM_QUANTITY_BIGNUM).round
```

Comment:
Shouldn't we use _ceil_ rather than _round_ ? If every quantity rounding is equivalent to _quantity.floor_ it might generate quantity overruns.

Reply:
We currently only provide a 1e-3 precision. below this limit a ceil will surely ensure the feasibility but may impeach some quantities combinations. Anyway, it is below our precision scope

The closest choice we have is rounding for time matrix in optimizer-ortools, we also chose a rounding over a ceil.

Reply:
I agree, to make sure, we might ned to do something but `ceil` is not a good idea, the issue is floating point precision. 

```ruby
irb(main):017:0> 2.011 * 1e3
=> 2011.0000000000002
irb(main):018:0> (2.011 * 1e3).ceil
=> 2012
```
Clearly we don't want 2012 here. 

I think you are misinterpreting the `round` , the`round` is there to prevent floating-point errors (and making sure we get 2011 for the above case). 

With Gwen's dev we are supporting up to 3 decimal places, so the rest of the decimals are dropped and as you said, we might generate violation if the user really uses more than 3 decimal places in their quantities/capacities -- but we can't guarantee what we don't support.

To use ceil, we need to do wizardry 
```ruby
irb(main):021:0> ((2.011 * 1e6).round / 1e3).ceil
=> 2011
```
but I am not sure if it worth the hassle.